Dockerfile.buildkit

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.16-alpine AS package
```

Comment:
is it possible to rename the file to `buildkit.Dockerfile`?
so any IDE/code editor would recognize this as a Dockerfile and we would have the intellisense, format and everything?

Reply:
Could you let me know if you are sure about this?
You asked me to rename it to `Dockerfile.buildkit` in a previous comment :smile: and after that VS Code is picking up the file format correctly.

Reply:
oh sorry, I got confused, you're right. the right pattern is `Dockerfile.somename`
thanks